### PR TITLE
Display created and updated timestamps in action sidebar

### PR DESCRIPTION
**Pull request by luketlancaster**
- [x] Add the Clubhouse Story ID: [ch20743]

### CODE REVIEW

File: src/js/apps/patients/sidebar/action-sidebar_app.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ export default App.extend({
   },
   onStart(options, activity) {
     this.activity = activity;
-    this.showView(new LayoutView({ model: this.action.clone() }));
+    const createdEvent = this.activity.filter({ type: 'ActionCreated' })[0];
```
Comment (paulfalgout) [inline src/js/apps/patients/sidebar/action-sidebar_app.js]: oh sorry `find` not `filter` will return the single value over the array

File: src/js/views/patients/sidebar/action-sidebar.hbs [hunk 2]
```diff
@@ -19,4 +19,9 @@
   <h3>Activity Log</h3>
   <div data-activity-region></div>
 </div>
+<hr>
```
Comment (paulfalgout) [inline src/js/views/patients/sidebar/action-sidebar.hbs]: no hr we'll add that to the class `action-sidebar__timestamps`

File: src/js/views/patients/sidebar/action-sidebar_views.js [hunk 3]
```diff
@@ -65,6 +65,7 @@ const LayoutView = View.extend({
   templateContext() {
     return {
       isNew: this.model.isNew(),
+      createdAt: this.options.createdEvent.get('date'),
```
Comment (paulfalgout) [inline src/js/views/patients/sidebar/action-sidebar_views.js]: let's use `this.getOption('createdEvent')`

File: test/integration/patients/sidebar/action-sidebar.js [hunk 4]
```diff
@@ -1,20 +1,35 @@
-import _ from 'underscore';
+import moment from 'moment';
 
 import { getResource } from 'helpers/json-api';
 
 context('action sidebar', function() {
-  specify('display action events', function() {
+  specify('display action sidebar', function() {
+    const now = moment();
```
Comment (paulfalgout) [inline test/integration/patients/sidebar/action-sidebar.js]: should be `moment.utc()` for data dates

File: src/js/views/patients/sidebar/action-sidebar-activity-views.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import hbs from 'handlebars-inline-precompile';
 import { View, CollectionView } from 'marionette';
 
-const AddedTemplate = hbs`
+const CreatedTemplate = hbs`
   {{#with metadata.editor}}
   {{formatHTMLMessage (intlGet "patients.sidebar.actionSidebar.activityViews.added") first_name = first_name last_name = last_name role = role.name}}
```
Comment (paulfalgout) [inline src/js/views/patients/sidebar/action-sidebar-activity-views.js]: might a well update the i18n key too

File: src/js/views/patients/sidebar/action-sidebar.hbs (hunk 2 above)
Comment (luketlancaster) [reply src/js/views/patients/sidebar/action-sidebar.hbs]: is that class correct?

**Pull request merged**